pages/upgrade-guide-2.x.md

```diff
@@ -196,6 +196,32 @@ QUnit.module( "router", {
 ```
 
 
+## Replace `assert.throws( block, string, message )` with `assert.throws( block, regexp, message )`
+
+The overload of `assert.throws()` which expected a block, error string, and assertion message has been removed and will now throw an exception. Use a regular expression instead.
```

Comment:
Not sure I follow- I do have the word throw in there. Or are you saying I need code tags around throw? If so, that seems unintuitive to me because I don't see code tags around words like return when used in the general sense (as opposed to referencing a specific API).